2020/26xxx/CVE-2020-26989.json

```diff
@@ -25,17 +25,17 @@
                                 }
```

Comment:
('Expecting property name enclosed in double quotes: line 103 column 13 (char 4261)',)